CHANGELOG.md

```diff
@@ -82,6 +82,10 @@
 - Segregated MMDS documentation in MMDS design documentation and MMDS user
   guide documentation.
 - The logger `level` field is now case-insensitive.
+- Disabled boot timer device after restoring a snapshot.
```

Comment:
These need to be part of v0.23. v0.22 is released from a branch and some CLs will need to be merged back to master. To make that effort easier I would just separate this from the rest.

Reply:
More of the entries here need to be split between v0.22 and v0.23/unreleased... I wouldn't burden this PR with that responsibility.